ngx-fudis/projects/ngx-fudis/src/lib/components/breadcrumbs/breadcrumbs.component.scss

```diff
@@ -0,0 +1,16 @@
+@use '../../foundations/colors/mixins.scss' as colors;
+@use '../../foundations/spacing/tokens.scss';
+@use '../../foundations/typography/mixins.scss' as typography;
+
+.fudis-breadcrumbs {
+	@include typography.body-text-md-regular;
+
+	display: flex;
+	align-items: center;
+}
+
+.fudis-separator {
```

Comment:
Remove unnecessary class due to `span` --> `fudis-icon` change.
If the class is still needed, it should follow BEM, _.fudis-breadcrumbs__separator_